src/AttitudeObserver.cpp

```diff
@@ -195,6 +268,98 @@ void AttitudeObserver::addToGUI(const mc_control::MCController & ctl,
                           reset(ctl);
                         }),
                  make_rpy_label("Result", m_orientation));
+
+  gui.addElement(category, ArrayLabel("Accelerometer", [this]() { return m_accIn; }),
+                 ArrayLabel("Gyrometer", [this]() { return m_rateIn; }));
+
+  if(m_gyroBiasFromMeasurement_running)
+  {
+    gui.addElement(category, mc_rtc::gui::ElementsStacking::Horizontal,
+                   ArrayLabel("Gyro Bias (calibrating) :", {"x", "y", "z"},
+                              [this]() -> Eigen::Vector3d { return (m_gyroMeasurementTotal / m_numberOfValues); }),
+                   Button("Stop\ncalibration", [this, &ctl, &gui, &category]() { setGyroBiasToMeanValue(); }));
+  }
+  else
+  {
+    gui.addElement(category, mc_rtc::gui::ElementsStacking::Horizontal, make_xyz_input("Gyro Bias:", m_gyrobias),
+                   Checkbox(
+                       "EKF bias\nestimation", [this]() { return withGyroBias_; },
+                       [this]() { switchWithGyroBias(!withGyroBias_); }),
+                   Button("Calibrate\nfrom data\n(static robot)",
```

Comment:
If I understood correctly, one needs to manually stop the calibration process? Isn't there a way to automatically detect when calibration is complete?